README.md

```diff
@@ -111,6 +111,15 @@ Returns an array of items of the trending items on the specified category.
   ```
 3. Create a `.env` file in the project root directory and add `WALMART_API_KEY=YOUR_API_KEY` to the file. This sets your api key as an environment variable.
 4. Test the project is working by running the example queries in your terminal:
```

Comment:
You should include AMAZON API KEY directions in .env like you stated in the PR.

Reply:
Thanks, I forgot to add that. https://github.com/sulaimonlasisi/clearance-app/pull/9/commits/33ad049c1a71bb41a11b56d7da0805c22e6e1ef5